src/main/java/net/bramp/ffmpeg/builder/FFmpegOutputBuilder.java

```diff
@@ -516,7 +517,7 @@ public EncodingOptions buildOptions() {
     if (targetSize > 0) {
       checkState(parent.inputs.size() == 1, "Target size does not support multiple inputs");
 
-      String firstInput = parent.inputs.get(0);
+      String firstInput = parent.inputs.iterator().next();
```

Comment:
If you later replace `List` by `Collection` the trick `.iterator().next()` still works